### PR TITLE
Cyril/report page ui cleanup

### PR DESCRIPTION
**Pull request by cyrielo**
This PR cleans up the report filter sidebar and makes the report table collapsible.

### CODE REVIEW

**Comment by cyrielo** [comment]
@cdebourcy PR has been updated

File: app/assets/javascripts/components/PipelineSampleReport.jsx [hunk 1]
```diff
@@ -52,13 +52,16 @@ class PipelineSampleReport extends React.Component {
         </span>
       );
     }
-    if (tax_info.tax_level == 1) {
-      // indent species rows
-      foo = <span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;{foo}</span>
+    if (tax_info.tax_level === 1) {
+      foo = <div className='specie-name'>{foo}</div>;
```
Comment (boris-dimitrov) [inline app/assets/javascripts/components/PipelineSampleReport.jsx]: Typo.   The word you want here is `species` not `specie`.

File: app/assets/javascripts/components/PipelineSampleReport.jsx [hunk 2]
```diff
@@ -108,6 +111,29 @@ class PipelineSampleReport extends React.Component {
...
+        ele.removeClass('fa-minus');
+        ele.addClass('fa-plus');
+      } else {
+        ele.removeClass('fa-plus');
+        ele.addClass('fa-minus');
+      }
+    });
+
+    // check if genus is toggle-able
+    const specieRow = $('.specie-name').parent().parent();
```
Comment (boris-dimitrov) [inline app/assets/javascripts/components/PipelineSampleReport.jsx]: Similarly, this is `species-name` not `specie-name`.

**Comment by boris-dimitrov** [comment]
After I click around on the same page for a bit, performance in Safari degrades to where I get the spinning ball for 10 seconds at a time and the browser is completely unresponsive when I try to collapse the "All taxa without genus classification" category.   Please debug the performance issue.

**Pull request merged**